flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/source/IcebergSource.java

```diff
@@ -86,7 +85,6 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@Experimental
```

Comment:
Do we want to say that this is `PublicEvolving` instead?

Reply:
I feel it is fine to be just public.